api/v1alpha1/orano2ims_types.go

```diff
@@ -64,8 +64,6 @@ type ResourceServerConfig struct {
 	//+optional
 	Extensions []string `json:"extensions,omitempty"`
 }
-
-// AlarmSubscriptionServerConfig contains the configuration for the alarm subscription server.
```

Comment:
Why are you removing this? It isn't a very informative comment, but doesn't hurt either.

Reply:
This caused PR sanity check failure.

Reply:
add back